client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -217,9 +217,12 @@ const renderBusinessExtensionHelpText = ( values, isInstallingActivating ) => {
 	}
 
 	const extensionsList = extensions
```

Comment:
It looks like maybe the `extension` keys are stored as `woocommerce-services=>shipping` and `woocommerce-services=>tax` instead of `woocommerce-services:shipping` and `woocommerce-services:tax` which is used by `pluginNames`. This may cause the problem I'm seeing.